pluginsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -1,150 +1,316 @@
 namespace Microsoft.Windows.DevHome.SDK
 {
-    [contractversion(1)] 
-    apicontract DevHomeContract {}
+    [contractversion(1)] apicontract DevHomeContract {}
 
-    [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 1)]
-    enum ProviderType
-    {
+        [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 1)] interface IExtension {
```

Comment:
I've never seen these definitions on the same line, and find it quite hard to read. Is this the new cool thing or can we change it back?

Reply:
So.  Yeah, It should not look like this.  But, this is what the auto-formatter does.  I don't want to fight it by making sure no one, ever, pressed ctrl+k then ctrl+d.

If we could figure out a way to make the format more readable, I'm down.

Reply:
Aha! Found it: https://stackoverflow.com/questions/74647010/formatting-attributes-on-the-same-line-in-editor-config

We can add a new section to the .editorconfig
```
[*.idl]
place_attribute_on_same_line = false
```

Reply:
Looks like this only works if using resharper.